### PR TITLE
Hifi signup

### PR DESCRIPTION
**Pull request by glm1550**

### CODE REVIEW

File: src/Navigation/StackNavigation.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const stackNavig = createNativeStackNavigator();
 /* Stack navigators push/pop screens from a navigation stack (similar to web browser) */
 export default function StackNavigation() {
   return (
-    <stackNavig.Navigator initialRouteName="Landing Page">
+    <stackNavig.Navigator initialRouteName="Log In">
```
Comment (curesh) [inline src/Navigation/StackNavigation.js:130]: Why is the inital route set to log in, instead of landing page

File: src/SignUp/LoginScreen.js [hunk 2]
```diff
@@ -1,49 +1,90 @@
...
     textAlignVertical: 'top',
-    backgroundColor: 'white',
+    backgroundColor: '#FFFFFA',
+  },
+
+  textInput: {
+    marginLeft: 7,
+    fontFamily: 'JosefinSans-SemiBold',
+    fontSize: 16,
+    // color: '#1D763C',
```
Comment (curesh) [inline src/SignUp/LoginScreen.js]: Remove unnecessary comments like this and similar ones below

File: src/SignUp/LoginScreen.js [hunk 3]
```diff
@@ -1,49 +1,90 @@
...
 
 const styles = StyleSheet.create({
+  containerInputs: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginTop: 200,
+
+  },
   container: {
```
Comment (curesh) [inline src/SignUp/LoginScreen.js]: I don't think this is used, can you remove it

File: src/SignUp/LoginScreen.js [hunk 4]
```diff
@@ -1,49 +1,90 @@
...
+
+  bottomTextBold: {
+    // marginLeft: 7,
+    fontFamily: 'JosefinSans-SemiBold',
+    fontSize: 16,
+    color: '#1D763C',
+
+  },
+
+  backgroundimage: {
```
Comment (curesh) [inline src/SignUp/LoginScreen.js]: Make this camelCase

File: src/SignUp/SignInScreen.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React, { useState, useRef } from 'react';
```
Comment (curesh) [inline src/SignUp/SignInScreen.js]: Can you rename this file to SignUpScreen instead of SignIn...

Comment (curesh) [reply src/SignUp/SignInScreen.js]: and the other one too

File: src/SignUp/LoginScreen.js [hunk 6]
```diff
@@ -58,22 +99,20 @@ export default function LoginScreen({ navigation }) {
...
+    <ImageBackground source={backgroundImage} resizeMode="cover" style={styles.backgroundimage}>
 
-    <View style={styles.container}>
-
-      <Image style={styles.image} source={logoImage} />
-
-      <View style={styles.container}>
+      <View style={styles.containerInputs}>
         <View style={styles.inputs}>
+          {/* <Icon icon="ant-design:user-outlined" /> */}
```
Comment (curesh) [inline src/SignUp/LoginScreen.js]: Remove excess comments

File: src/SignUp/SignInScreen.js [hunk 7]
```diff
@@ -28,67 +29,95 @@ const styles = StyleSheet.create({
     flex: 1,
     justifyContent: 'center',
     alignItems: 'center',
-    backgroundColor: '#FFFFFF',
+    // backgroundColor: '#FFFFFF',
```
Comment (curesh) [inline src/SignUp/SignInScreen.js]: remove
comment

**Review by curesh — CHANGES_REQUESTED** [review]
Hi Grace this feature is gorgeous thank for your hard work uwu
I added a few syntax, code cleanliness comments but otherwise looks good. Feel free to merge it after you fix those issues. Great stuff!

**Comment by glm1550** [comment]
Hifi version of signup. Signup feature not yet implemented - covered in future sprint.

**Pull request merged**